docker/Dockerfile.rhel8-2.oc4-4

```diff
@@ -55,6 +55,10 @@ RUN rm /etc/rhsm-host && \
     echo "/usr/local/lib" > /etc/ld.so.conf.d/local-lib.conf && \
     echo "/usr/local/lib64" >> /etc/ld.so.conf.d/local-lib.conf
 
+# wait for it
```

Comment:
@lionelgo 

can you please also apply the same changes to : 

- `docker/Dockerfile.rhel8` file ---> It is the one used by CI
- `docker/Dockerfile.centos7/8` files  (do we want to keep them?)

Reply:
Done, we can keep Dockerfile.centos8  for the time being.